assets/Icons/icons.tsx

```diff
@@ -0,0 +1,199 @@
+export const CheckmarkIcon = () => (
```

Comment:
I think you should keep a KDM icon folder so all icons are organized into their own folders